### PR TITLE
[WIP] Adding controller class for the landing pages, and added basic HTML pages using Bootstrap

### PR DESCRIPTION
**Pull request by ryanribeiro**
## Proposed changes

> Describe the big picture of your changes. If it fixes a bug or resolves a feature request, be sure to link to that issue

## Types of changes

> What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] The title is described the contents of the PR
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (where applicable)
- [ ] I have moved issue to the “Need review” columns of projects

## Known issues

> Create issue for known issues and link them in the pull request

Closes #[issue]

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -10,12 +10,12 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel>
-      <module name="app" target="11" />
+      <module name="app" target="1.8" />
```
Comment (yeskunall) [inline .idea/compiler.xml]: This is a big no. Remove any changes to the `.idea` folder.

File: .idea/modules.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app.iml" filepath="$PROJECT_DIR$/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app.iml" filepath="$PROJECT_DIR$/app.iml" />
```
Comment (yeskunall) [inline .idea/modules.xml]: Why is this here twice? Like I said before, remove any changes to the `.idea` folder.

File: pom.xml [hunk 3]
```diff
@@ -1,28 +1,51 @@
...
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-rest</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
```
Comment (yeskunall) [inline pom.xml]: We’re not using `h2`, we’ll be using MySQL. Remove any references to any db stuff right now. I’ll be pushing some changes soon enough that will introduce a DB layer for __pproval.app__.

File: src/main/java/com/pproval/app/Application.java [hunk 4]
```diff
@@ -1,11 +1,94 @@
+//package com.pproval.app;
```
Comment (yeskunall) [inline src/main/java/com/pproval/app/Application.java]: What is this?

File: src/main/java/com/pproval/app/Application.java [hunk 5]
```diff
@@ -1,11 +1,94 @@
...
+    public static void main(String[] args) {
+            SpringApplication.run(Application.class,args);
+    }
+
+
+    @Bean
+    public CommandLineRunner demo(ArticleRepository articleRepo, UserRepository userRepo){
+        return (args) -> {
+            //SET UP USERS IN DATABASE
+            Submitter submit = new Submitter("Robyn","Van Gool");
```
Comment (yeskunall) [inline src/main/java/com/pproval/app/Application.java]: Keep this for testing. We can make the same queries with Hibernate + MySQL.

**Comment by yeskunall** [comment]
@RobynVG @PragyaJollySingh update the PR template according to the changes you’ve made. Michael will ask to see proposed changes. 😅

File: .idea/compiler.xml (hunk 1 above)
Comment (ryanribeiro) [reply .idea/compiler.xml]: Resolved since I reverted .idea

File: .idea/modules.xml (hunk 2 above)
Comment (ryanribeiro) [reply .idea/modules.xml]: Resolved since I reverted .idea

File: pom.xml (hunk 3 above)
Comment (ryanribeiro) [reply pom.xml]: Resolved since I reverted .idea

Comment (ryanribeiro) [reply pom.xml]: For this iteration we have to leave user persistence in there as we do not have the time to implement MySQL before the deadline.

File: src/main/java/com/pproval/app/Application.java (hunk 5 above)
Comment (ryanribeiro) [reply src/main/java/com/pproval/app/Application.java]: It has been commented out since then and we will revisit using it at a later time.

File: src/main/resources/templates/submitTest.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+      <a class="nav-item nav-link active" href="/home">Home <span class="sr-only">(current)</span></a>
+      <a class="nav-item nav-link" href="/submitTest">Submit Article</a>
+      <a class="nav-item nav-link" href="#">View Submitted</a>
+      <a class="nav-item nav-link disabled" href="#">Disabled</a>
+    </div>
+  </div>
+</nav>
+
+<body>
+  <h1>Please let this work</h1>
```
Comment (yeskunall) [inline src/main/resources/templates/submitTest.html]: I’m sorry but I just had to post this: LMAO.

I feel you tho @ryanribeiro. 😅
